### ISSUE TITLE
Some opcodes disable wild surges

### ISSUE THREAD

**Issue by lynxlynxlynx** [gemrb/gemrb#942]
To quote Bubb:
> Did you know that there are some spells that completely bypass the effects of Wild Magic?
> 
> This is due to the engine harcoding some opcodes to, when present, automatically cause Wild Surge rolls to be bypassed. These opcodes are as follows:
> 
> #32 (0x20) Cure: Death (Raise Dead)
> #192 (0xC0) Spell Effect: Find Familiar
> #214 (0xD6) Spell Effect: Select Spell
> #232 (0xE8) Spell Effect: Cast Spell on Condition
> #234 (0xEA) Spell Effect: Contingency Creation
> #256 (0x100) Spell: Spell Sequencer Active
> #257 (0x101) Spell: Spell Sequencer Creation
> 
> Some of the more popular spells that use these opcodes include Fire Shield (Red), Fire Shield (Blue), and Spell Immunity. So, have fun casting these 4th and 5th level spells via Nahal's with no risk whatsoever! 

So pretty lame! Perhaps it would work if we put the vanilla spells into splspec.2da with 4 set for flags, so mods can disable this cheese and we don't need to hardcode it. Some of them already have SF_HLA or SF_TRIGGER spell flags set and can thus be ignored.

**Comment by burner1024** [comment]
Sequencers and contingencies don't surge when being cast, but then [the next spell will surge, 100%](https://pihwiki.bgforge.net/Baldur%27s_Gate:_Wild_Mage_Compendium#Dweomering_sequencers_and_contingencies), and that is cumulative.

**Comment by lynxlynxlynx** [comment]
No, per the description, any opcode from the list will save the whole spell. So at most the payload when the condition is met would surge (eg. if you're in a wild magic area).

**Comment by burner1024** [comment]
Yes, I tried fire shields, everytime "wild surge: spell cast normally". So that's kind of cheese.
